apps/ui/src/views/My/Home.vue

```diff
@@ -77,26 +78,20 @@ async function handleEndReached() {
 
 onMounted(() => {
   metaStore.fetchBlock(networkId.value);
-  loadFollows(networkId.value);
 });
 
 watch(
-  () => web3.value.account,
-  () => loadFollows(networkId.value)
-);
-
-watch(
-  () => follows.value,
+  () => followedSpacesStore.followedSpacesIds,
   () => {
     loaded.value = false;
     proposals.value = [];
 
-    if (!follows.value.length) {
+    if (!followedSpacesStore.followedSpacesIds.length) {
```

Comment:
nit. as this value is what we are watching I think we should use it from watcher function arguments instead of externally. It should be also a bit cleaner without looking into store directly.

```js
(followedSpacesIds) => {
  if (!followedSpacesIds.length) {}
  // ...
}
```